toolchain/mfc/user_guide.py

```
@@ -0,0 +1,664 @@
+    cons.print("Available templates: [cyan]1D_minimal[/cyan], [cyan]2D_minimal[/cyan], [cyan]3D_minimal[/cyan]")
+    cons.print("[dim]Or use 'example:<name>' to copy from examples[/dim]")
+    cons.print()
+
+    name = Prompt.ask("Case name", default="my_case")
+    template = Prompt.ask("Template", default="1D_minimal")
+
+    cmd = f"./mfc.sh new {name} -t {template}"
+    cons.print()
+    cons.print(f"[dim]Running: {cmd}[/dim]")
+    cons.print()
+    os.system(cmd)
```

Ruff (0.14.14)

[error] Starting a process with a shell, possible injection detected

(S605)